### PR TITLE
Refs 214: fix readme mistake

### PR DESCRIPTION
**Pull request by jlsherrill**
start:beta doesn't actually work
and i need a commit to try the -build push

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ sudo npm run patch:hosts
 
 1. `npm install`
 
-2. `PROXY=true npm run start:beta`
+2. `PROXY=true npm run start`
 
 3. Open browser in URL listed in the terminal output
```
Comment (swadeley) [inline README.md:23]:
```suggestion
3. With a browser, open URL listed in the terminal output
```

**Review by swadeley — COMMENTED** [review]
lgtm

**Review by swadeley — APPROVED** [review]

**Comment by jlsherrill** [comment]
https://issues.redhat.com/browse/HMSCONTENT-214

**Comment by jlsherrill** [comment]
⚠️ This task currently requires qe-approval, but this PR does not fully resolve the task.  Please contact QE to determine appropriate testing required.

**Pull request merged**